README.md

```diff
@@ -29,15 +29,15 @@ To quickly get a flavor of MC<sup>2</sup>, you can work in a Docker image that c
 
 1. You must have [Docker](https://docs.docker.com/get-docker/) installed. We recommend giving Docker at least 2 CPUs, 6 GB of memory, and 2 GB of swap space (instructions for [Mac](https://docs.docker.com/docker-for-mac/#resources), [Windows](https://docs.docker.com/docker-for-windows/#resources)). Without sufficient resources, the quickstart may not work.
 
-    Once that is done, pull the Docker image, launch a container, and start an SSH server inside the container. Note that you only need to start the SSH server if running in a Docker container -- most cloud VMs automatically start an SSH server on boot.
-
+    Once that is done, pull the Docker image and launch a container.
     ```sh
     docker pull mc2project/mc2
-
     docker run -it -p 22:22 -p 50051-50055:50051-50055 -w /root mc2project/mc2
+    ```
+    Start an SSH server inside the container. (Note that you only need to start the SSH server if running in a Docker container -- most cloud VMs automatically start an SSH server on boot.)
```

Comment:
I'd emphasize this more, something like "*Docker specific instruction:*" or "*If you're running in Docker:*"